tar/bsdtar.h

```diff
@@ -35,6 +35,10 @@
 #include <stdint.h>
 #include <stdio.h>
 
+#include "elasticarray.h"
+
+ELASTICARRAY_DECL(TAPENAMES_SETUP, tapenames_setup, char *);
```

Comment:
I would call this STRLIST.  More generally, elastic arrays should have names appropriate to what they contain, not what they're used for.